src/Parser/AfflictionWarlock/Modules/WarlockCore/getDamageBonus.js

```diff
@@ -0,0 +1,3 @@
+export default function getDamageBonus(event, increase) {
```

Comment:
So basically something you use in the function where you calculate healing effectivity? Instead of just event.amount I should add event.amount + event.absorbed and work with that? Makes sense, I didn't think of it.